### ISSUE TITLE
Live channel incompatible

### ISSUE THREAD

**Issue by fildy** [Fleker/CumulusTV#128]
Hi, 

Somehow my channel list from cumulus (m3u8) doesn't recognize by the live channel. Previously it worked. 
Then I try to add vivo channel from cumulus, and this channel is recognized by the live channel.
Is there anything that cause this issue?

**Comment by Fleker** [comment]
Is your m3u8 file a playlist or one stream? Directly adding a playlist
won't work. You need to open the file using CumulusTV and import the
channels.

**Comment by josuearisty** [comment]
Hey nick, I have a question about loading a .json file.

For some unknown reason I cant load my .json file onto another nexus player. It seems to be a problem with gdrive on the np.

Is it there another way to load? 

What about making an option to load the .json file from the np itself. By searching on internal storage?

Thanks for all!

**Comment by Fleker** [comment]
Internal storage is an idea. I'm planning on making the syncing mechanics
more abstract in the next big version.

Cam you determine why Drive isn't working correctly on your NP?

**Comment by fildy** [comment]
One stream and I do add via cumulus. Before everything was ok for months. But now, it's not working. 

When I grab, it has been recognized, but cannot being load by the live channel.
In the cumulus, my channel is working perfectly.

Any clue?
![image](https://cloud.githubusercontent.com/assets/6156088/15993996/52df0072-30f7-11e6-9eaa-08ec6b9c8dfb.png)
![image](https://cloud.githubusercontent.com/assets/6156088/15993997/52fdb878-30f7-11e6-84b8-b42f5b518e92.jpeg)

**Comment by josuearisty** [comment]
Well, a created a .json channel list for my brotherands NP because he has an iphone, then I copied the file to his gdrive.

Then with cumulustv on np I cant import from gdrive, Im able to see the gdrive window but I dont know why I cant select the file. I created a folder to test and I can not select the folder either.

Thats why I say that problem comes with gdrive. His np is updated to lastest montly update. And all apps are updated too.
